Added all dependencies for Mana Toolkit.

adde88 opened the pull request:
Tagging you as requested: @Foxtrot 

Mostly python-related dependencies required to make sslstrip+ and dns2proxy work.

Makefiles for hostapd-mana and asleap will be commited in a seperate PR.

adde88 committed:
Added all dependencies for Mana Toolkit. Mostly python-related dependencies required to make sslstrip+ and dns2proxy work.

adde88 committed:
Removed the python packages

adde88 committed:
Added hcxdumptool and hcxtools. Needed to perform the PMKID attack on the Wifi Pineapples. And needed to work on the outputted files.

adde88 commented:
@foxtrot

I've committed the Makefiles for hcxdumptool and hcxtools.

foxtrot commented on network/scapy/Makefile:
```diff
@@ -0,0 +1,54 @@
+#
```
Is there anything extra added in this scapy package? Otherwise, I would prefer to use the one from the new OpenWRT feeds.

Thanks!

adde88 replied:
OpenWRT seems to have an older version (2.3.1-1).
I've just been trying to keep it updated, as quite a few changes and improvements have been made to it since then.

So basicly it's just a newer release.
Just use OpenWRT's version if you feel like it. 

Changelog:
https://github.com/secdev/scapy/releases

adde88 committed:
Removed Scapy

adde88 committed:
Updated hcxdumptool and hcxtools Makefile

foxtrot commented on network/dns2proxy/Makefile:
```diff
@@ -0,0 +1,51 @@
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/LeonardoNve/dns2proxy.git
```
Please use https://codeload.github.com/x for the package sources, in all Makefiles that pull from GitHub.

adde88 replied:
Some git repos didn't tad the releases.
So i've edited those Makefiles to use the source locally within a subdir.

adde88 committed:
hostapd-mana:

Added Makefile and source for hostapd-mana
This version does NOT contain any postinst scripts,and will only build the binaries needed to launch and control hostapd-mana, as well as all the needed configs and certificates.
Some improvements have also been made. So this version differs from "hostpad-2.6.5-13", which can be found here: https://github.com/adde88/hostapd-mana
I've added support for all EAP encryption methods, and other "small tweaks" to hostapd's .config file.

Signed-off-by: Andreas Nilsen <adde88@gmail.com>

adde88 committed:
I've edited the Makefiles. Some git repos didn't tag the releases.
So that's why i've added the source to the packages. (dns2proxy, sslstrip2, asleap).
I have also corrected the licence of the Makefiles, and added license variables to hostapd-mana.

Signed-off-by: Andreas Nilsen <adde88@gmail.com>

foxtrot reviewed (changes requested):
There seems to be a lot of files in this commit. A lot of those are binaries (.o, .d and ELF).

Please 'make clean' inside your package folders and re-commit without toolchains and binaries.

adde88 committed:
Remove accidently added build files and directories.

Signed-off-by: Andreas Nilsen <adde88@gmail.com>

foxtrot commented on network/hostapd-mana/files/.svn/entries:
```diff
@@ -0,0 +1,368 @@
+10
```
Are these .svn files necessary? Seems like bloat from the barrier breaker days.

foxtrot replied:
Other than that, everything is looking good I think.

adde88 replied:
I've removed them. Shouldn't be needed at all.

adde88 committed:
hostapd-mana: removed .svn directory. Bloat from the old days.

Signed-off-by: Andreas Nilsen <adde88@gmail.com>

adde88 committed:
Merge branch 'master' of https://github.com/hak5/pineapple-packages-community

adde88 committed:
hostapd-mana: Makefile update, config tweaks, etc.

- Updated the Makefile, removed bloat.
- Renamed directories from "mana-toolkit" to "hostapd-mana", as this package package should not build a toolkit.
- I've also thoroughly gone over all the config files that gets built, and added some stuff to them. Some configs have been removed,as they're not longer needed.

Tested on both WiFi Pineapple NANO and TETRA, running latest firmware (2.6.0) with OpenWRT 19.07

Signed-off-by: Andreas Nilsen <adde88@gmail.com>

bruvv commented:
bump
